python310Packages.qcs-api-client: 0.21.2 -> 0.21.3

fabaff opened the pull request:
###### Description of changes

<!--
For package updates please link to a changelog or describe changes, this helps your fellow maintainers discover breaking updates.
For new packages please briefly describe the package or provide a link to its homepage.
-->

###### Things done

<!-- Please check what applies. Note that these are not hard requirements but merely serve as information for reviewers. -->

- Built on platform(s)
  - [ ] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.05 Release Notes (or backporting 22.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2305-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

<!--
To help with the large amounts of pull requests, we would appreciate your
reviews of other pull requests, especially simple package updates. Just leave a
comment describing what you have tested in the relevant package/service.
Reviewing helps to reduce the average time-to-merge for everyone.
Thanks a lot if you do!

List of open PRs: https://github.com/NixOS/nixpkgs/pulls
Reviewing guidelines: https://nixos.org/manual/nixpkgs/unstable/#chap-reviewing-contributions
-->

fabaff committed:
python310Packages.qcs-api-client: add changelog to meta

fabaff committed:
python310Packages.qcs-api-client: 0.21.2 -> 0.21.3

Diff: https://github.com/rigetti/qcs-api-client-python/compare/refs/tags/v0.21.2...v0.21.3

Changelog: https://github.com/rigetti/qcs-api-client-python/releases/tag/v0.21.3

fabaff commented:
Result of `nixpkgs-review pr 213124` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>7 packages failed to build:</summary>
  <ul>
    <li>python310Packages.cirq</li>
    <li>python310Packages.cirq-rigetti</li>
    <li>python310Packages.pyquil</li>
    <li>python311Packages.cirq</li>
    <li>python311Packages.cirq-rigetti</li>
    <li>python311Packages.pyquil</li>
    <li>python311Packages.qcs-api-client</li>
  </ul>
</details>
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>python310Packages.qcs-api-client</li>
  </ul>
</details>

fabaff committed:
python310Packages.pyquil: add chagnelog to meta

fabaff commented:
Depends on #213082.

NickCao commented:
Result of `nixpkgs-review pr 213124` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>4 packages failed to build:</summary>
  <ul>
    <li>python311Packages.cirq</li>
    <li>python311Packages.cirq-rigetti</li>
    <li>python311Packages.pyquil</li>
    <li>python311Packages.qcs-api-client</li>
  </ul>
</details>
<details>
  <summary>4 packages built:</summary>
  <ul>
    <li>python310Packages.cirq</li>
    <li>python310Packages.cirq-rigetti</li>
    <li>python310Packages.pyquil</li>
    <li>python310Packages.qcs-api-client</li>
  </ul>
</details>

fabaff committed:
python311Packages.qcs-api-client: disable tests on Python 3.11

fabaff committed:
python310Packages.pyquil: 3.3.2 -> 3.3.3

Diff: https://github.com/rigetti/pyquil/compare/refs/tags/v3.3.2...v3.3.3

Changelog: https://github.com/rigetti/pyquil/blob/v3.3.3/CHANGELOG.md

fabaff commented:
Result of `nixpkgs-review pr 213124` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>2 packages failed to build:</summary>
  <ul>
    <li>python311Packages.cirq</li>
    <li>python311Packages.cirq-rigetti</li>
  </ul>
</details>
<details>
  <summary>6 packages built:</summary>
  <ul>
    <li>python310Packages.cirq</li>
    <li>python310Packages.cirq-rigetti</li>
    <li>python310Packages.pyquil</li>
    <li>python310Packages.qcs-api-client</li>
    <li>python311Packages.pyquil</li>
    <li>python311Packages.qcs-api-client</li>
  </ul>
</details>
